src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/QNameFinder.java

```diff
@@ -42,8 +45,11 @@ public QNameFinder( final ReadMetadata metadata,
         }
         if ( intervalsIndex >= intervalsSize ) return noName;
         final SVInterval indexedInterval = intervals.get(intervalsIndex);
-        final SVInterval readInterval = new SVInterval(readContigId, readStart, read.getUnclippedEnd()+1);
-        if ( indexedInterval.isDisjointFrom(readInterval) ) return noName;
+        final SVInterval unclippedReadInterval = new SVInterval(readContigId, read.getUnclippedStart(), read.getUnclippedEnd());
+        final SVInterval clippedReadInterval = new SVInterval(readContigId, read.getStart(), read.getEnd());
```

Comment:
Make no interval before its time.  Move this line below the next one.

Reply:
done